src/dune_rules/coq/coq_lib.ml

```diff
@@ -458,23 +458,23 @@ module DB = struct
       Resolve.Memo.List.map theories ~f
     ;;
 
-    let create_from_stanza_impl (coq_db, db, dir, (s : Coq_stanza.Theory.t)) =
+    let create_from_coq_package_impl (coq_db, db, dir, (s : Coq_package.meta)) =
```

Comment:
Our initial idea was to convert a `Coq_package` into a `Coq_stanza` but some fields don't seem easy to populate, so I am trying the other way around. However this memo table relies on physical equality of `Coq_stanza` so I'm not sure this still works as expected.

Reply:
I see! This is indeed not needed for the db, I'd suggest to make it an option for now, there are already some codepaths where we use get_exn for it.